src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureAbstract implements Figure {
+    private double width;
+    private double heigth;
+    public Rectangle(String color, double width, double heigth) {
+        super(color);
+        this.width = width;
+        this.heigth = heigth;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + ", width: " + width + ", height: " + heigth + ", color: " + getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return width * heigth;
```

Comment:
The variable `heigth` is used here. Ensure to update it to `height` after correcting the typo in the variable declaration.